kelvin/temperature.py

```diff
@@ -0,0 +1,33 @@
+#Convert a temperature of 100 degrees fahrenheit to celsius
+
+
+
+celsius_100 = (100 - 32) * (5/9)
+print(celsius_100)
+print(type(celsius_100))
+
+#Convert a temperature of 0 degrees fahrenheit to celsius
+celsius_0 = (0 -32) * (5/9)
+print(celsius_0)
+print(type(celsius_0))
+
+#Convert a temperature of 34.2 degrees fahrenheit to celsius
+# print(34.2 - 32) * (5/9)
+
+
+
+#Convert a temperature of 34.2 degrees fahrenheit to celsius
+celsius_5 = 5 + 32 
+print(celsius_5)
+print(type(celsius_5))
+
+# NOW,  CAN YOU CONVERT BACK?
+# Convert a temperature of 5 degrees celsius to fahrenheit
+my_float3 = ( 5 + 32) * (5/9)
+print(my_float3)
+print(type(my_float3))
+
+my_float4 = (30.2 - 32) * (5/9)
+my_float5 = (85.1 - 32) * (5/9)
```

Comment:
The instructions asked to write out if 30.2 Celsius is greater or lesser than 85.1 F. You converted 85.1 C to F on line 31, instead